helm/charts/hpe-csi-driver/questions.yml

```diff
@@ -31,3 +31,9 @@ questions:
   required: false
   description: "Specify password for iSCSI CHAP authentication"
   group: "HPE iSCSI settings"
+- variable: registryPrefix
+  label: "registry"
+  type: string
+  default: "quay.io"
+  description: "Specify registry prefix for HPE CSI Images"
+  group: "HPE CSI Driver settings"
```

Comment:
- Use "Specify registry prefix for CSI images" (we use this verbiage above)
- Please use a newline at EOL